node/api.md

```diff
@@ -19,11 +19,11 @@
     * [/network/tx/:id](#networktxid)
 * [Wallet API](#wallet-api)
     * [/wallet/new](#walletnew)
-    * [/wallet/:id/balance](#walletidbalance)
-    * [/wallet/:id/txs](#walletidtxs)
-    * [/wallet/:id/address](#walletidaddress)
-    * [/wallet/:id/receiver](#walletidreceiver)
-    * [/wallet/:id/buildtx](#walletidbuildtx)
+    * [/wallet/balance](#walletidbalance)
```

Comment:
the link now becomes (and others similarly)
```suggestion
    * [/wallet/balance](#walletbalance)
```

Reply:
The commit above already fixed it.